frontend/components/notifications/management/NotificationsManagementDashboards.vue

```diff
@@ -256,7 +259,22 @@ const handleDelete = (payload: Parameters<typeof deleteDashboardNotifications>[0
           :header="$t('notifications.col.dashboard')"
         >
           <template #body="slotProps">
-            <span>
+            <BcTooltip
+              v-if="slotProps.data.is_archived"
+              tooltip-width="320px"
```

Comment:
this might suffice: (😪 applying it by default would be best)
```suggestion
             fit-content
```

Reply:
Done